.pre-commit-config.yaml

```
@@ -1357,6 +1357,13 @@ repos:
         pass_filenames: false
         files: ^airflow-core/src/airflow/api_fastapi/execution_api/.*\.py$
         require_serial: true
+      - id: generate-airflowctl-datamodels
+        name: Generate Datamodels for AirflowCTL
+        language: python
+        entry: uv run -p 3.12 --no-dev --no-progress --active --group codegen --project apache-airflow-ctl --directory airflow-ctl/ datamodel-codegen
+        pass_filenames: false
+        files: ^airflow-core/src/airflow/api_fastapi/core_api/.*\.py$
+        require_serial: true
```

Could you please add the same things for auth datamodels?
```suggestion
        require_serial: true
- id: generate-airflowctl-auth-datamodels
        name: Generate Datamodels for AirflowCTL
        language: python
        entry: uv run -p 3.12 --no-dev --no-progress --active --group codegen --project apache-airflow-ctl --directory airflow-ctl/ datamodel-codegen --input=../airflow-core/src/airflow/api_fastapi/auth/managers/simple/openapi/v1-generated.yaml --output=src/airflowctl/api/datamodels/auth_generated.py
        pass_filenames: false
        files: ^airflow-core/src/airflow/api_fastapi/auth/managers/simple/datamodels/.*\.py$
        require_serial: true
```

---

Added under the same pre-commit step